azure-pipelines.yml

```diff
@@ -282,44 +259,6 @@ jobs:
       includePatterns:
         - "/hw/top_earlgrey/Vchip_earlgrey_verilator"
 
-# Software targeting the English Breakfast top level is produced by patching
-# the source tree before building. This builds a selected subset of software
-# only.
-# TODO: This is a rather ugly hack, which will go away once we properly support
-# building more than one top-level design with different parametrizations.
-# Work towards this goal is tracked in issue #4669.
-- job: build_and_execute_verilated_tests_englishbreakfast
-  displayName: Verilated English Breakfast (Execute)
-  # Build and execute tests on the Verilated English Breakfast toplevel design with Bazel
-  pool:
-    vmImage: ubuntu-20.04
-  dependsOn: chip_englishbreakfast_verilator
-  steps:
-  - template: ci/checkout-template.yml
-  - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_englishbreakfast_verilator
-  - template: ci/load-bazel-cache-write-creds.yml
```

Comment:
We might want to have Bazel cache setup properly?

Reply:
Yes, probably. I don't think I have access to GitHub Actions secrets, so I'll bring this up next week

Reply:
We can use workload identity federation, although this would need adding permission on the bucket.

Reply:
Can you use WIF? The docs only discuss service accounts and HTTP auth: https://bazel.build/remote/caching#cloud-storage

Reply:
`--google_default_credentials` would work

Reply:
We decided to do this in a follow up and not block this PR